web/static/css/home.css

```diff
@@ -270,6 +280,52 @@ h1 {
 .about-askbywho__logo img {
   width: 130px;
 }
+.about-askbywho__share {
+  text-align: center;
+  justify-content: center;
+  font-weight: 600;
+}
+.about-askbywho__share img {
+  width: 52px;
+  padding-right: 5px;
+}
+.about-askbywho__share a {
+   text-decoration: none;
```

Comment:
Having an issue with a very persistent link text-decoration... @ohanhi could you give a look?